bedrock/products/templates/products/vpn/landing-refresh.html

```diff
@@ -83,13 +95,13 @@ <h1>{{ ftl('vpn-landing-powerful-privacy-for-peace') }}</h1>
       <p class="c-main-cta">
         <a class="mzp-c-button mzp-t-product mzp-t-xl" href="#pricing" data-cta-type="button" data-cta-text="Save on Mozilla VPN" data-cta-position="primary">
           {# Remove these conditionals once cybersecurity month banner is removed see issue 13761 #}
```

Comment:
Suggestion: update this comment to reflect the GitHub issue for this new banner?